File: src/FizzBuzz/FizzBuzz.java
package FizzBuzz;

public class FizzBuzz {
    protected String value;

    public FizzBuzz(int number){
        if (number % 3 == 0 && number % 5 == 0){
            value = "FIZZBUZZ";
        }
        else if (number % 3 == 0){
            value = "FIZZ";
        } else if (number % 5 == 0){
            value = "BUZZ";
        }
    }
}

File: src/Python/FizzBuzz/FizzBuzz.py

class FizzBuzz(object):

    def __init__(self, number: int):
        self.__value = ''
        if number % 3 == 0 and number % 5 == 0:
            self.__value = 'FIZZBUZZ'
        elif number % 3 == 0:
            self.__value = 'FIZZ'
        elif number % 5 == 0:
            self.__value = 'BUZZ'
        else:
            self.__value = str(number)

    @property
    def value(self) -> str:
        return self.__value
